CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 
 ## [5.1.2] - 2023-06-23
+### Fixed
+- [#111](https://github.com/reportportal/agent-js-playwright/issues/111) Test is not finished when `expect().toPass()` exceed test timeout
```

Comment:
This fix will be added to the next version, so please move it to the top of the file.